subsys/net/lib/zperf/zperf_tcp_receiver.c

```diff
@@ -100,10 +98,50 @@ static void tcp_received(const struct sockaddr *addr, size_t datalen)
 	}
 }
 
+static int tcp_bind_listen_connection(struct zsock_pollfd *pollfd,
+				      struct sockaddr *address)
+{
+	uint16_t port;
+	int ret;
+
+	if (address->sa_family == AF_INET) {
+		port = ntohs(net_sin(address)->sin_port);
+	} else {
+		port = ntohs(net_sin6(address)->sin6_port);
+	}
+
+	ret = zsock_bind(pollfd->fd, address, sizeof(*address));
+	if (ret < 0) {
+		NET_ERR("Cannot bind IPv%d TCP port %d (%d)\n",
```

Comment:
Please fix other log messages as well, as I see there are more places like this - `\n` is not needed with the logger, as it adds new line implicitly.